src/tools/merssh/command.cpp

```diff
@@ -223,11 +226,31 @@ bool Command::isValid() const
     return true;
 }
 
+static QString readRelativeRoot()
```

Comment:
I meant static member, sorry for that.

It would be nice to flatten the body using this style:

```
if (!cond)
    return;

....
```

Reply:
fixed